documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc

```diff
@@ -161,9 +161,7 @@ If a customer MAC address is set to ff:ff:ff:ff:ff:ff, then it’s ignored.
 
 === Device-specific private key
 
-NOTE: This is not yet implemented for BCM2712 devices
-
-Eight rows of OTP (256 bits) are available for use as a device-specific private key. This is intended to support file-system encryption.
+Non-2712 devices have 8 rows of OTP (256 bits) available for use as a device-specific private key. On 2712 devices there are 16 rows of OTP data (512 bits). This is intended to support file-system encryption.
```

Comment:
```suggestion
Devices that use the Broadcom BCM2712 processor have 16 rows of OTP data (512 bits) to support filesystem encryption.
Devices that do not use BCM2712 have 8 rows of OTP (256 bits) available for use as a device-specific private key.
```
Suggestion: make it clearer that we're referring to the Broadcom BCM2712 processor.

Reply:
_technically_ we could release a future device with a processor older than BCM2712, so I don't know if this necessarily holds up. Maybe just "Raspberry Pi 5" and "prior to Raspberry Pi 5"? But there are enough specifics here related to the CPU that this is presumably subject to change in future processors.


...definitely worth a revisit in the future when the situation changes. For now, I'm happy enough with it.